generate-stackbrew-library.sh

```diff
@@ -2,8 +2,8 @@
 set -Eeuo pipefail
 
 declare -A release_channel=(
-	[stable]='18.0.6'
-	[production]='17.0.7'
+	[stable]='19.0.1'
+	[production]='19.0.1'
```

Comment:
the production channel is still on 17, you can see here that 17 instances only get the `17.0.8` update and not `19.0.1`
https://github.com/nextcloud/updater_server/blob/aad2440bcec4af6d19c3cae5993868dea8d8da64/config/config.php#L140-L142
```suggestion
	[production]='17.0.8'
```

Reply:
Fixed (see comment above)